### ISSUE TITLE
Failed when using the format IP:port to set desired_capabilities

### ISSUE THREAD

**Issue by cookie314** [robotframework/SeleniumLibrary#206]
When using S2L, if I want to set a manual proxy, I set the parameter desired_capabilities like this:  desired_capabilities=proxyType:manual , httpProxy:IP:port
Then it failed because the IP:port has a ":", which affect the resolving of this json.
Log show:
ValueError: too many values to unpack
Traceback (most recent call last):
"C:\RobotFramework\Python2.7.3\lib\site-packages\robotframework_selenium2library-1.2.0-py2.7.egg\Selenium2Library\keywords_browsermanagement.py", line 473, in _create_desired_capabilities (key, value) = cap.split(":")

Besides, when I set the path of profile I need to use ":" and "\", like "c:\profiles\", still fail. So maybe I used wrong syntax. Could you please help on this, thx.

**Comment by ombre42** [comment]
Duplicate of [proxy support is still missing in 1.0.1](https://github.com/rtomac/robotframework-selenium2library/issues/90)
I think this will be fixed in 1.3 by allowing a dictionary to passed in for desired_capabilities, but I have not tested it.

**Comment by ymost** [comment]
I had the same issue.

Until an official I fix comes a long I just changed the line:

``` python
(key, value) = cap.split(":")
```

to:

``` python
(key, value) = cap.split(":", 1)
```

In the method '_create_desired_capabilities' in the file '_browsermanagement.py'.

Works like a charm.

**Comment by emanlove** [comment]
Fixed in #252.
